src/win/thread.c

```diff
@@ -95,6 +95,13 @@ int uv_thread_create(uv_thread_t *tid, void (*entry)(void *arg), void *arg) {
   return uv_thread_create_ex(tid, &params, entry, arg);
 }
 
+
+int uv_thread_detach(uv_thread_t *tid) {
+  CloseHandle(*tid);
```

Comment:
CloseHandle can fail; for example, if you call uv_pthread_detach twice on the same handle.

There's a general discrepancy in that e.g. glibc's pthread_detach returns EINVAL when you call it on an already detached thread, while musl reports no error. I guess that can't be helped, just pointing it out.

Reply:
True that, I've added error handling to this one.